webapp/src/services/movie/MovieService.tsx

```diff
@@ -0,0 +1,21 @@
+import { Logger } from 'simple-logging-system';
+import MovieApi from '../../api/movie/MovieApi';
+
+const logger: Logger = new Logger('MovieService');
+
+export default class MovieService {
+  constructor(private readonly movieApi: MovieApi) {
+  }
+
+  getMoviesByGenre = (id: number) => this.movieApi.getTopOfMoviesByGenre(id)
+    .catch((error) => {
+      logger.error('Error', { id, err: error });
```

Comment:
un peu plus explicite l'erreur haha.
```suggestion
      logger.error('Failed to fetch movies for genre with id: {} ',  id, { error });
```
Dans les logger en JS (dans la lib Plume), les `{}` sont remplacés les uns après les autres par les arguments que tu passes à la méthode de log. Le dernier argument étant l'erreur.